spec/yardstick/v2_client/test_centre_time_window_spec.rb

```diff
@@ -11,4 +11,36 @@
       expect(subject.upcoming_and_recent(token)).to eq([])
     end
   end
+
+  describe :proctors do
+    it 'create the right type of proctors' do
+      stub_path_response(:get, '/v2/sittings/83/proctors', 200, [{
+          id: 53,
+          first_name: 'Bruce',
+          last_name: 'Wayne',
+          email: 'doyouwanttoknowmysecretidentity@wayneenterprises.com',
+          type: 'AdminUser',
+          roles: ['proctor']
+        }, {
+          id: 82,
+          first_name: 'Clark',
+          last_name: 'Kent',
+          email: 'ckent@dailyplanet.com',
+          type: 'Proctor'
+        }, {
+          id: 38,
+          first_name: 'Lois',
+          last_name: 'Lane',
+          email: 'lois@dailyplanet.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.